documentation-site/examples/notification/overrides.js

```diff
@@ -12,7 +12,7 @@ export default () => (
         }),
       },
       CloseIcon: {
-        component: DeleteAlt,
+        component: props => <DeleteAlt {...props} />,
```

Comment:
Here's another instance of overrides needing to change because of flow check fail

Reply:
Why do we need to wrap it tho? Does it mean that types are incorrect?